lib/index.js

```diff
@@ -124,13 +124,28 @@ bedrock.events.on(
  * @param identity the Identity containing at least the minimum required data.
  * @param [meta] optional meta information to include.
  *
- * @return a Promise that resolves to the new record.
+ * @return {Promise} the new record.
  */
 api.insert = brCallbackify(async ({actor, identity, meta}) => {
   assert.object(identity, 'identity');
   assert.string(identity.id, 'identity.id');
   assert.optionalString(identity.owner, 'identity.owner');
 
+  // select only allowed identity properties. It is particularly important that
+  // `sysResoureRole` is excluded in case someone upgrades to v5.x+ without
+  // updating their code.
```

Comment:
Should the code check for sysResourceRole and throw an error if present?  (or just use that assert lib used above?)

Reply:
I thought of throwing.  But seems like a shame to do an extra check from here to eternity for something that won't exist in the common case.  Permission checks will be completely broken in bad user code with the current patch, so people will get the message that something is wrong.